### DISCUSSION TITLE
Concurrent Events

### DISCUSSION THREAD

**Discussion by baggytrowsers** [cpvalente/ontime#459] [Ideas]
Sorry if this is deviating from the purpose of the software...

We've been using ontime to display relevant tinformation for our high school exams. It generally works well, but our main issue is that we need to run different exams at the same time, but with different display text and end times to be displayed currently.

e.g.
- Exam 1: Starts at 1pm finishes at 3pm
- Exam 2: Starts at 1pm finishes at 4pm

When we start the event at 1pm, we would like to display both events with the different end times.

Is that something that is possible?

**Comment by cpvalente** [comment]
Hi, no need to apologise, I am happy that users get involved and thank you for doing so.

I am unsure I understand the request correctly. 

Are these displayed in the same screen or different screens? In the latter case, can the [countdown view ](https://ontime.gitbook.io/v2/views/countdown)help?

There is nothing that I can see that would stop ontime from doing this. It might just be that we don't have a correct view for the information you need.
Would you be able to provide some more context and a small prototype and I can see what can be done from my side? I would love to help

**Comment by baggytrowsers** [comment]
It would need to be displayed on the same screen. Even if I ran two separate coundown views in separate side-by-side windows, I would still have the issue that I can only trigger one event. I need to run multiple events concurrently.

The view would essentially show the start and end times of multiple events (with different lengths).
